src/main/scala/com/fulcrumgenomics/util/LogDouble.scala

```diff
@@ -169,4 +175,6 @@ class LogDouble(val logValue: Double) extends AnyVal with Ordered[LogDouble] {
     else if (a < b) throw new IllegalArgumentException("Subtraction will be less than zero.")
     else a + log1p(0.0 - exp(b - a)) // using log1p, otherwise it would be `a + log(1.0 - exp(b - a))`
   }
+
+  override def toString: String = s"$linearValue.to ($logValue)"
```

Comment:
@tfenne I have no clue what we want here for a string, but this is clearly wrong.